### ISSUE TITLE
RKCB you show 1 or 4 and partner signs off, but you have 4

### ISSUE THREAD

**Issue by Muppet1** [EdwardPiwowar/BBA#277]
![image](https://github.com/EdwardPiwowar/BBA/assets/11272970/989ff5d4-f1e0-44b5-b120-ac473aadd8ad)
3DE7A6060698B4BF418BEF148023

team mates auction.  Should they have bid on?

**Comment by ThorvaldAagaard** [comment]
I have an agreement, that (3,4 or 5) always bid on, and that you never can ask without aces, and normally you only ask if you can handle the answer

**Comment by ThePokerDude** [comment]
Agree.I weould like to see a general approach implemented, which could look like this:

If RKCB is bid and the answer to RKCB has an equivocal meaning (0 or 3 / 1 or 4) and the player who asked signs off
(bids trump below slam level as answer to the RKCB response), the answerer (to RKCB) is obliged to bid on at least once.

Equivalent for NT situations.

**Comment by EdwardPiwowar** [comment]
This approach is partially implemented but has many limitations because it is too often unsuccessful.
![image](https://github.com/EdwardPiwowar/BBA/assets/150140020/a9ddd2b6-7311-465a-9843-14b7a3bb834c)
![image](https://github.com/EdwardPiwowar/BBA/assets/150140020/75740f9a-ea9e-42ae-8d79-96ce43ae70b6)

In the current BBA, E is bidding 7H.
![image](https://github.com/EdwardPiwowar/BBA/assets/150140020/032c65ba-e847-4981-b3cd-391f0dd080aa)

**Comment by ThePokerDude** [comment]
8140 fails
![image](https://github.com/user-attachments/assets/ffe902f2-4f90-4a14-9b69-240a99cb258b)

**Comment by EdwardPiwowar** [comment]
The rules (3,4 or 5) always bid are logically fine, but they don't always work.
0329339BAA2375104C33438EB8D1
works
![image](https://github.com/user-attachments/assets/d2843574-6454-4058-abe5-c318a03824d8)

032C339BAA2375104C33438EB8D1
doesn't work (AC<=>AD)
![image](https://github.com/user-attachments/assets/6c5956d2-544b-4c41-a2d8-1e8172478e4e)

Need for an additional rule.

**Comment by ThePokerDude** [comment]
Still not fixed. West must bid 6h after 5h to show it's 4 and not 1 keycard 
![image](https://github.com/user-attachments/assets/9ad4ddaa-c851-4cd5-b2ed-2731db550519)
